src/spiceypy/spiceypy.py

```diff
@@ -10326,13 +10326,18 @@ def pckcov(pck: str, idcode: int, cover: SpiceCell) -> None:
 
     :param pck: Name of PCK file.
     :param idcode: Class ID code of PCK reference frame.
-    :param cover: Window giving coverage in pck for idcode.
+    :param cover: Optional SPICE Window giving coverage in "pck" for "idcode".
+    :return: coverage window for a specified "idcode" in specified "pck" file
     """
     pck = stypes.string_to_char_p(pck)
     idcode = ctypes.c_int(idcode)
-    assert isinstance(cover, stypes.SpiceCell)
-    assert cover.dtype == 1
+    if cover is None:
+        cover = stypes.SPICEDOUBLE_CELL(20000)
```

Comment:
I believe scard needs to be called on the cell, see https://naif.jpl.nasa.gov/pub/naif/misc/toolkit_docs_N0067/C/cspice/pckcov_c.html

Reply:
interesting... anyways thanks for adding that I feel better about it now heh